examples/AdvancedOperations/GetAdGroupBidModifiers.php

```diff
@@ -126,18 +135,53 @@ public static function runExample(
         // the ad group bid modifier in each row.
         foreach ($response->iterateAllElements() as $googleAdsRow) {
             /** @var GoogleAdsRow $googleAdsRow */
+            $AdGroupBidModifier = $googleAdsRow->getAdGroupBidModifier();
```

Comment:
Use the lower-case variable. (I guess this will fail phpcs?)

Reply:
Good catch! It does not make phpcs fail.